Add document explaining re-auth when apple dev account is migrated (#1343)
home-assistant/companion.home-assistant: docs/troubleshooting/ios-developer-account-migration.md

---
title: iOS developer account migration
id: 'ios-developer-account-migration'
---

The Home Assistant ![iOS](/assets/iOS.svg)app will move from the Nabu Casa Apple Developer account to the Open Home Foundation Apple Developer account.

> ## What this means for you

Reviewer: ```suggestion
## What developer account migration of the app means for you
```

Author: `What app developer account migration means for you` ?

Reviewer: oh, yes. Much better :raised_hands:

> After the migration is complete, you may need to sign in again in the iOS app for each Home Assistant server you have configured.

Reviewer: ```suggestion
After the Home Assistant app has migrated from the Nabu Casa Apple Developer account to the Open Home Foundation Apple Developer account, you may need to sign in again in the iOS app for each Home Assistant server you have configured.
```
I know this is duplication... But I thought maybe reiterating this here could be helpful. I stumbled here because I thought it was about my account. So I was wondering, which account? Do I have a developer account for Apple?

Author: Good point 👍🏻 making it clear it's the app (company/foundation/owner) developer account

This can happen because iOS stores app credentials in Apple's keychain, and keychain data is tied to the app's Apple Developer account. When an app moves to a different developer account, previously stored credentials are not available to the new app identity.

In practice, this means you may be asked to enter your Home Assistant username and password again, server by server.

## TestFlight users

During the account migration, the current TestFlight channel will be closed.

After the migration is complete, a new TestFlight invite link will be published.

We will announce the new link on our social channels and in the documentation.

> ## How to prepare before migration

Reviewer: ```suggestion
## Preparing migration
```

Author: `Preparing for the migration` maybe? Otherwise it sounds like the user is preparing the migration itself

Reviewer: yes, that sounds better.
